src/sensors/icm20948sensor.h

```diff
@@ -38,25 +38,35 @@ class ICM20948Sensor : public Sensor
     }
 
     void motionLoop() override final;
-
     void sendData() override final;
     void startCalibration(int calibrationType) override final;
-    void save_bias(bool repeat);
-    void load_bias();
 
 private:
-    void calculateAcceleration(Quat *quaternion);
+    void calculateAccelerationWithoutGravity(Quat *quaternion);
     unsigned long lastData = 0;
+    unsigned long lastDataSent = 0;
     int bias_save_counter = 0;
     bool newTap;
     int16_t rawAccel[3];
+    double dmpNumberToDoubleConverter = 1073741824.0;
```

Comment:
having this in a define would be preferred as this will cause a memory access but a define won't